inst/rmarkdown/templates/xaringan/resources/metropolis.css

```diff
@@ -41,17 +41,31 @@ position: absolute;
   font-size: 35px;
   text-align: left;
   margin-left: 00px;
+  margin-bottom: -10px;
 }
 
-.remark-slide-content > h3 {
+.remark-slide-content > h3, .pull-left > h3, .pull-right > h3 {
   padding-top: -15px;
-  padding-bottom: 10px;
+  padding-bottom: 1px;
   color: #1A292C;
   text-shadow: none;
   font-weight: 400;
   font-size: 30px;
   text-align: left;
   margin-left: 00px;
+  margin-bottom: -10px;
+}
+
+.remark-slide-content > h4, .pull-left > h4, .pull-right > h4 {
+  padding-top: -15px;
+  padding-bottom: 1px;
+  color: #1A292C;
+  text-shadow: none;
+  font-weight: 400;
+  font-size: 30px;
+  text-align: left;
+  margin-left: 00px;
+  margin-bottom: -10px;
```

Comment:
Can these rules be merged to above? I see there are several repetitions.

Reply:
does this apply to you? :smile: no CSS expert